scripts/promote.sh

```diff
@@ -1,7 +1,6 @@
 #!/bin/bash
 packages=(
   @patternfly/patternfly
-  @patternfly/react-catalog-view-extension
```

Comment:
I actually don't think this needs to be deleted if the react-catalog-view-extension is still going to be published in the patternfly npm org. I can still use this script to check whether or not the catalog view has any new pre-release candidates when i promote each sprint